dist/focus-ring.js

```diff
@@ -265,7 +225,6 @@ document.addEventListener('DOMContentLoaded', function() {
     if (index(el).contains('focus-ring'))
       return;
     index(el).add('focus-ring');
-    el.setAttribute('data-focus-ring-added', '');
```

Comment:
What's a use case for an author-added `focus-ring` class? And _why_ would we want to support that if this is a polyfill for `:focus-ring` and there would be no similar pattern once this hopefully is implemented in browsers. Wouldn't we want to discourage authors adding `focus-ring`?